bears/python/YapfBear.py

```diff
@@ -0,0 +1,135 @@
+import sys
+
+from coalib.bears.LocalBear import LocalBear
+from yapf.yapflib.yapf_api import FormatFile
+from coalib.results.Result import Result
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.results.Diff import Diff
+from coalib.misc.ContextManagers import prepare_file
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+class YapfBear(LocalBear):
+    """
+    Check and correct formatting of Python Code using ``yapf`` utility.
+
+    See <https://github.com/google/yapf> for more information.
+    """
+    LANGUAGES = {"Python"}
+    AUTHORS = {'The coala developers'}
+    REQUIREMENTS = {PipRequirement('yapf', '0.10')}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+
+    def run(self, filename, file,
+            max_line_length: int=79,
+            tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+            allow_multiline_lambdas: bool=False,
+            blank_line_before_nested_class_or_def: bool=False,
+            coalesce_brackets: bool=False,  # not yet supported by yapf
```

Comment:
if this isn't supported, don't provide the option yet and rather make a comment down in the source with a TODO